contrib/drivers/huawei/dorado/metrics.go

```diff
@@ -0,0 +1,311 @@
+// Copyright (c) 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dorado
+
+import (
+	"github.com/satori/go.uuid"
+	"strconv"
+	"time"
+
+	log "github.com/golang/glog"
+	. "github.com/opensds/opensds/contrib/drivers/utils/config"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/opensds/opensds/pkg/utils/config"
+	"gopkg.in/yaml.v2"
+)
+
+/*
+Naming Map:
+metrics             --> OceanStor
+iops                --> Throughput(IOPS)(IO/s)
+bandwidth           --> Bandwidth(MB/s) / Block Bandwidth(MB/s)
+latency             --> Average I/O Latency(us)
+service_time        --> Service Time(Excluding Queue Time)(ms)
+cache_hit_ratio     --> % Hit
+cpu_usage           --> CPU Usage(%)
+*/
+// Todo: Move this Yaml config to a file
+// Todo: Add resources for "volume", "disk" and "port".
+var data = `
+resources:
+  - resource: pool
+    metrics:
+      - iops
+      - bandwidth
+      - latency
+      - service_time
+      - utilization_prcnt
+    units:
+      - tps
+      - mbs
+      - microsecond
+      - ms
+      - prcnt
+  - resource: controller
+    metrics:
+      - iops
+      - bandwidth
+      - latency
+      - service_time
+      - cache_hit_ratio
+      - cpu_usage
+    units:
+      - tps
+      - mbs
+      - microsecond
+      - ms
+      - prcnt
```

Comment:
I think it's duplicated here.

Reply:
no, one for controller and one for pool , they are not the same